jmx/src/main/java/org/jboss/as/jmx/JMXSubsystemAdd.java

```diff
@@ -72,6 +76,18 @@ static void launchServices(OperationContext context, ModelNode model, ManagedAud
         } else {
             isMasterHc = false;
         }
+        if (context.getProcessType() == ProcessType.HOST_CONTROLLER) {
```

Comment:
Yes this was for DOMAIN_SERVER not HC (i did something wrong with cleaning the code)